src/store/modules/chatMessages.js

```diff
@@ -36,47 +36,61 @@ const getters = {
 // actions
 const actions = {
   initializeChatMessages({ dispatch, commit, rootGetters }) {
-    const conversationId = "VC01";
-    let directLineMessageRecievedHandler = (userName, message) => {
+    let directLineMessageRecievedHandler = (userName, message, convoId) => {
+      if (!state.chatConversation.some((chat) => chat.id === convoId)) {
+        commit("addChatConversation", {
```

Comment:
It's not a good idea to create chat conversations on message sent. It sends us further down the rabbit hole of being locked into having an init send message

Reply:
Can get conversation id by subscribing to connectionStatus and responding to a status of 2 meaning connected